### DISCUSSION TITLE
Windows-10 Start context menu

### DISCUSSION THREAD

**Discussion by HubKing** [valinet/ExplorerPatcher#833] [General]
The menu items seem too crammed, even when that of real Windows 10, if I remember it correctly. Compare the spacing with that of Win11 context menu. I think it needs to have bigger font and more spacing (match that of Win11 menu).

![image](https://user-images.githubusercontent.com/10900489/153414517-cadf35e3-a5b8-48b3-a9ae-8081707b70ca.png)

**Comment by valinet** [comment]
The menu is skinned by Windows’ internal feature. It is functional and utilitarian, the looks are decent on unaltered configurations, so I think it is fine. In your screenshot, the spacing is less than it is on regular setups.

**Comment by HubKing** [comment]
It turned out that this is because I had turned off Properties -> System tray -> Skin taskbar and tray pop-up menus. I thought I would get native look by doing so, but it seems it does not, because I looked at the context menu on a computer running Windows 10, and it was not as crammed like that. So, in short, the "skinned" Start button context menu of Explorer Patcher actually looks closer to that of real Windows 10.

**Reply by Poopooracoocoo** [reply]
(now you can mark your solution/answer)

**Reply by valinet** [reply]
Options do certain effects, some people might prefer, some might not. The default config is usually pretty close to stock Windows 10.
